assets/css/editor.scss

```diff
@@ -60,3 +60,26 @@
 	margin: 0;
 	padding: 0;
 }
+
+.theme-twentytwenty {
+	.wp-block {
+		h2.wc-block-grid__product-title {
+			@include font-size(large);
+		}
+
+		.wc-block-grid__product-price,
```

Comment:
Is `.wc-block-grid__product-price` needed here? It didn't seem to make any difference on my end, but maybe I'm missing something?

Reply:
I am not sure why I added this in the first place, but after removing it, the styles remained the same.